crates/ecstore/Cargo.toml

```diff
@@ -101,6 +101,13 @@ aws-credential-types = { workspace = true }
 aws-smithy-types = { workspace = true }
 parking_lot = { workspace = true }
 moka = { workspace = true }
+serde_urlencoded.workspace = true
+google-cloud-storage = "1.1.0"
+google-cloud-auth = "1.0.1"
+azure_core = "0.29.1"
+azure_storage = "0.21.0"
```

Comment:
Version management of crates·moved to the root directory `cargo.toml` for unified management